composer.json

```diff
@@ -75,7 +77,8 @@
     "scripts": {
         "auto-scripts": {
             "cache:clear": "symfony-cmd",
-            "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd"
+            "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd",
+            "assets:install %PUBLIC_DIR%": "symfony-cmd"
```

Comment:
having both commands does not make sense. The second one will override the first installation that used symlinks.

Reply:
Yes, this came from here: https://github.com/symfony/recipes/blob/cb1cb50ace8fe990b60a7ea9e8f840cd6c30cbfb/symfony/framework-bundle/5.4/manifest.json#L12

I'm going to remove our customized command based on symlinks, to better align with Symfony's defaults.